src/api/forms/service.js

```diff
@@ -164,10 +171,6 @@ export async function createLiveFromDraft(formId, author) {
   // Get the form metadata from the db
   const form = await getForm(formId)
 
-  if (!form) {
-    throw Boom.notFound(`Form with id '${formId}' not found`)
-  }
-
   if (!form.draft) {
     throw Boom.badRequest(`Form with id '${formId}' has no draft state`)
```

Comment:
We've started adding new errors with ~`id`~ `ID` so shall we update these old ones?

```suggestion
    throw Boom.badRequest(`Form with ID '${formId}' has no draft state`)
```

Reply:
Done